src/commonMain/kotlin/exchange.dydx.abacus/protocols/ParserProtocol.kt

```diff
@@ -64,10 +65,12 @@ inline fun <reified T> ParserProtocol.asTypedList(list: Any?): List<T>? {
                     }
                     json.decodeFromString<T>(itemString)
                 } catch (e: SerializationException) {
-                    Logger.e { "Failed to parse item: $item" }
+                    val typeClassifier = typeOf<T>().classifier
```

Comment:
Reflection is slow, let's avoid if possible.

Since `T` is `reified` here, you should be able to use `T::class`